src/System.Devices.I2c/II2cDeviceStatics.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Devices.I2c
+{
+    // TODO: Need to review this.
+    internal interface II2cDeviceStatics
```

Comment:
Why an empty interface?

Reply:
I added and missed by mistake.  It was another based on Windows.Devices and appeared to include methods related to AQS to find items.  I planned to review the need, but will removed for now.